sign-up/sign-up.js

```diff
@@ -0,0 +1,33 @@
+import { createParticipant, getWorkshops } from '../fetch-utils.js';
+
+const form = document.querySelector('.sign-up-form');
+
+form.addEventListener('submit', async e => {
+    e.preventDefault();
+    const data = new FormData(form);
+
+    await createParticipant({
+        name: data.get('name'),
+        email: data.get('email'),
+        workshop_id: data.get('workshop-id')
+    });
+
+    form.reset();
+
```

Comment:
you could add a `window.location.replace` here to redirect back to the workshops list page once the new participant is created, if you want that.